functions/tips/index.js

```diff
@@ -94,8 +94,15 @@ const httpsAgent = new https.Agent({keepAlive: true});
  */
 exports.connectionPooling = async (req, res) => {
   try {
-    const {data} = await fetch('/', {httpAgent, httpsAgent});
-    res.status(200).send(`Data: ${data}`);
+    const url = 'https://www.example.com/';
```

Comment:
Can we add a TODO here? A developer has to replace this URL.

Reply:
Not necessarily - the URL provided will work as-is.

That said, I'll add an optional TODO.